login.php

```diff
@@ -0,0 +1,20 @@
+<?php
+session_start ();
+$session = $_SESSION ['logged-in'];
+
+if ($session === 1) {
+    header("Location: cms.php");
+} elseif ( $session === 2) {
```

Comment:
Ok so if you are reusing the variable `$session` then you could keep it.. I like to just use `$_SESSION['logged-in']`.
up to you I guess